superset/views/core.py

```diff
@@ -298,6 +299,65 @@ class DatabaseAsync(DatabaseView):
 appbuilder.add_view_no_menu(DatabaseAsync)
 
 
+class CsvToDatabaseView(SimpleFormView):
+    form = CsvToDatabaseForm
+    form_title = _('CSV to Database configuration')
+    add_columns = ['database', 'schema', 'table_name']
+
+    def form_get(self, form):
+        form.sep.data = ','
+        form.header.data = 0
+        form.squeeze.data = False
+        form.names.data = None
+        form.mangle_dupe_cols.data = True
+        form.skipinitialspace.data = False
+        form.skip_blank_lines.data = True
+        form.parse_dates.data = True
+        form.infer_datetime_format.data = True
+        form.dayfirst.data = False
+        form.decimal.data = '.'
+        form.error_bad_lines.data = False
+        form.if_exists.data = 'append'
+        all_datasources = db.session.query(
```

Comment:
this one may have to stay here since you don't want to do it in module scope...

Reply:
fixed